.circleci/config.yml

```diff
@@ -113,6 +113,7 @@ jobs:
               --environment=$ENV \
               --tag=$TAG \
               --wait=300
```

Comment:
I think you'll need a trailing backslash to avoid a syntax problem. Otherwise looks right.

Reply:
Thanks! Updating it now.